cli/functionary/environment.py

```diff
@@ -0,0 +1,98 @@
+import json
+
+import click
+import requests
+
+from .config import get_config_value, save_config_value
+
+
+def get_environment_list():
+    """
+    Helper function to get the environment list from host
+
+    Args:
+        None
+
+    Returns:
+        None
+
+    Raises:
+        ClickException if bad get request
+    """
+    token = get_config_value("token")
+    teams_url = get_config_value("host") + "/api/v1/teams"
+    header = {"Authorization": f"Token {token}"}
+    response = requests.get(teams_url, headers=header)
+
+    if response.ok:
+        data = json.loads(response.text).get("results")
+
+        env_list = []
+        for team in data:
+            for env_set in team.get("environments"):
+                env_list.append(env_set)
+        return env_list
```

Comment:
You'll want to capture the team name along with the environment in some way here.  Without it, you lose some important context when you display the list.  Here's what my output looks like when I run `functionary environment set`:

```shell
$ functionary environment set
Available Environments:
    1) default
    2) dev
    3) prod
    4) default
Select environment:
```

Without the team name, there's no way to differentiate the environments that happen to have the same name.